.travis.yml

```diff
@@ -18,23 +18,9 @@ before_script:
 - carthage bootstrap --platform ios
 
 script:
-- set -o pipefail && xcodebuild -project "SmartDeviceLink-iOS.xcodeproj" -scheme "SmartDeviceLink" -sdk "iphonesimulator13.2" -destination "OS=13.2,name=iPhone 11" -configuration Debug ONLY_ACTIVE_ARCH=NO RUN_CLANG_STATIC_ANALYZER=NO GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES ENABLE_TESTABILITY=YES test | xcpretty -c;
-- set -o pipefail && xcodebuild -project "SmartDeviceLink-iOS.xcodeproj" -scheme "SmartDeviceLink-Example-ObjC" -sdk "iphonesimulator13.2" -destination "OS=13.2,name=iPhone 11" -configuration Debug ONLY_ACTIVE_ARCH=NO build | xcpretty -c;
-- set -o pipefail && xcodebuild -project "SmartDeviceLink-iOS.xcodeproj" -scheme "SmartDeviceLink-Example-Swift" -sdk "iphonesimulator13.2" -destination "OS=13.2,name=iPhone 11" -configuration Debug ONLY_ACTIVE_ARCH=NO build | xcpretty -c;
+- set -o pipefail && xcodebuild -project "SmartDeviceLink-iOS.xcodeproj" -scheme "SmartDeviceLink-Example-ObjC" -destination "platform=iOS Simulator,OS=13.2.2,name=iPhone 11" -configuration Debug ONLY_ACTIVE_ARCH=NO RUN_CLANG_STATIC_ANALYZER=NO build | xcpretty -c;
```

Comment:
I am seeing these warnings:
```
*** Skipped downloading ios-snapshot-test-case.framework binary due to the error:
858	"Maximum number of login attempts exceeded. Please try again later."
859*** xcodebuild output can be found in /var/folders/17/5mc7816d3mndxjqgplq6057w0000gn/T/carthage-xcodebuild.ST78ry.log
860*** Skipped downloading Nimble.framework binary due to the error:
861	"Maximum number of login attempts exceeded. Please try again later."
862*** Skipped downloading ocmock.framework binary due to the error:
863	"Maximum number of login attempts exceeded. Please try again later."
864*** Skipped downloading Quick.framework binary due to the error:
865	"Maximum number of login attempts exceeded. Please try again later."
866
```

Since these frameworks are downloaded on every build it looks like we might be able to cut down on build times by caching:
https://github.com/travis-ci/travis-ci/issues/6348#issuecomment-292463940

Reply:
Done